frontend/my-app/src/Pages/Results.tsx

```diff
@@ -26,16 +28,154 @@ const Results: React.FC = () => {
+            alignItems="center"
+            spacing={2}
+            sx={{
+              display: 'flex',
+              flexWarp: 'wrap',
+              backgroundColor: 'primary.main',
+              mt: 10,
+              borderRadius: '40px',
+              width: '600px',
+              height: '600px',
+              px: 1,
+              pb: 2
+            }}>
+              <Paper
+                sx={{
+                  mt: 3,
+                  width: '550px',
+                  height: '344px',
+                  // TODO: get image of restaurant instad of placeholder. Image could be photo or google maps static map
+                  backgroundImage: 'url("https://www.richaven.com/wp-content/uploads/2012/10/uw-ee-cse-11.jpg")',
+                  borderRadius: '10px',
+                  backgroundSize: 'cover',
+                  border: 2
+                }}>
+                <Stack
+                  justifyContent="flex-start"
+                  alignItems="center"
+                  spacing={2}
+                  sx={{
+                    display: 'flex',
+                    flexWarp: 'wrap'
+                  }}>
+                  <Paper
+                    sx={{
+                      mt: 2,
+                      mx: 2,
+                      p: 2,
+                      borderRadius: '10px',
+                      border: 2
+                    }}>
+                    <Typography
+                      variant="h5"
+                      component="h2"
+                      textAlign="center">
+                      {result.name}
+                    </Typography>
+                  </Paper>
+                </Stack>
+              </Paper>
+              <Typography // TODO: get description of restaurant, can omit
+                  variant="body1"
+                  textAlign="center"
+                  color="text.secondary">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                    incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+                    exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+                    dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+                    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+                    anim id est laborum.
+                </Typography>
+              <Paper
+                sx={{
+                  mt: 2,
+                  mx: 2,
+                  p: 2,
+                  borderRadius: '10px',
+                  backgroundColor: 'success.main'
+                }}>
+                <Typography
+                  variant="h6"
+                  component="h2"
+                  textAlign="center">
+                  Average Travel Time: {result.travelTimes.reduce((a, b) => a + b, 0) / result.travelTimes.length}
```

Comment:
Something to consider with the group - did we want to display average or actual travel times for each person?